### ISSUE TITLE
Add Main Icon to Windows

### ISSUE THREAD

**Issue by emoon** [emoon/ProDBG#216]
As there is a new icon for ProDBG it would be good to have the Windows version include it (currently only in Mac version) it may be that minifb needs to be updated to support it.

**Comment by Jake-Shadle** [comment]
Not sure if this issue means what I think it means, but here is an example of linking an icon into a Rust Windows executable https://github.com/klutzy/rust-windows/tree/master/examples

**Comment by emoon** [comment]
Yeah the task is pretty much to get a icon on the Windows version which isn't the default one. I think Tundra supports compilation of resources files so linking that one it should be fine but likely need to add some things to minifb so it can be set on window creation also.
